### PR TITLE
Add System management page

### PR DESCRIPTION
**Pull request by chrisrohr**

### CODE REVIEW

File: service/src/main/java/org/kiwiproject/champagne/model/DeployableSystem.java [hunk 1]
```diff
@@ -37,12 +37,26 @@ public class DeployableSystem {
     @With
     List<SystemUser> users;
 
+    /**
+     * Transitive property that is the resolved development environment name.
```
Comment (sleberknight) [inline service/src/main/java/org/kiwiproject/champagne/model/DeployableSystem.java:41]: What do you mean by "Transitive property"?

**Review by sleberknight — APPROVED** [review]

File: service/src/main/java/org/kiwiproject/champagne/model/DeployableSystem.java (hunk 1 above)
Comment (chrisrohr) [reply service/src/main/java/org/kiwiproject/champagne/model/DeployableSystem.java:41]: Meaning it is not stored directly in the database. I augment the record with extra data that goes in those extra fields

Comment (sleberknight) [reply service/src/main/java/org/kiwiproject/champagne/model/DeployableSystem.java:41]: OK, so then what it should really say is _transient_ right?

Comment (chrisrohr) [reply service/src/main/java/org/kiwiproject/champagne/model/DeployableSystem.java]: Yep, fixed and pushed

**Review by sleberknight — APPROVED** [review]

**Pull request merged**